modules/contract/contracts.js

```diff
@@ -125,18 +144,25 @@ class Contracts {
 
         amount = amount | 0;
       
+        Logger.warn('please wait....this could take a while to deploy contract on the network');
         try {
             const contract = fs.readFileSync(contractPath, 'utf8');
             const result = await conseiljs.TezosNodeWriter.sendContractOriginationOperation(
                                       tezosNode, keystore, amount*1000000, undefined,
                                       100000, '', 10000, 500000, 
-                                      contract, initValue, conseiljs.TezosParameterFormat.Michelson);         
+                                      contract, initValue, conseiljs.TezosParameterFormat.Michelson);        
+                                      
+            if(!tezosNode.startsWith('http://localhost')) {
```

Comment:
you can check for localhost only.
what if someone has given https://localhost as provider?